README.md

```diff
@@ -293,6 +293,10 @@ If you are interesed in libraries MxEngine depend on, consider reading [dependen
 <i>shadow casting from dynamic lights, screen-space reflections</i>
 </p>
 
+### GodRay effect by [#Fall2019](https://github.com/fall2019)
```

Comment:
I appreciate putting contributors name in the project readme, but suggest a small fix:

have: 
```
### Special thanks
#### God Ray Effect by [#Fall2019](https://github.com/fall2019)
<image>
```

Reply:
also I suggets making a screenshot without editor and window toolbar (on top)